### PR TITLE
Spelling code

### PR DESCRIPTION
**Pull request by jsoref**
##### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

##### Test Plan

The misspellings have been reported at https://github.com/jsoref/netdata/commit/33c5b20ed9511dac0a23ee916fa247cc2f388304#commitcomment-45444734

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/netdata/commit/e2737c090e4b1d7b1bce405b9f702bcf526852a3

##### Additional Information

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

This is split from #10428

And is now split into the items below. (This PR will be closable imminently.)

### CODE REVIEW

**Comment by ilyam8** [comment]
@jsoref if you move all https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin changes to a separate PR i can review it asap. Could you do it?

**Comment by jsoref** [comment]
@ilyam8: I think that should do it. (sorry about the multiple pushes)

**Comment by ilyam8** [comment]
>  I think that should do it.

@jsoref i mean only the `collectors/python.d.plugin/**` files. This PR changes a lot of files from a lot of areas.

File: aclk/legacy/agent_cloud_link.h [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 #define ACLK_MAX_TOPIC 255
 
-#define ACLK_RECONNECT_DELAY 1 // reconnect delay -- with backoff stragegy fow now
+#define ACLK_RECONNECT_DELAY 1 // reconnect delay -- with backoff strategy fow now
```
Comment (underhood) [inline aclk/legacy/agent_cloud_link.h:27]: fow->for

**Review by underhood — CHANGES_REQUESTED** [review]

**Review by vlvkobal — DISMISSED** [review]

**Comment by ilyam8** [comment]
@jsoref thanks for your amazing work, we really appreciate it. It is a pity that it take so long to get it merged.

I think, to speed up the process, we can split the PR per area, so we have less reviewers in every PR.

This is the list of files with the changes:

```cmd
0 ~ $ curl   -s -H "Accept: application/vnd.github.v3+json"   https://api.github.com/repos/netdata/netdata/pulls/10521/files | jq ".[] | .filename" | sort
"Dockerfile.test"
"aclk/legacy/aclk_common.c"
"aclk/legacy/aclk_lws_wss_client.c"
"aclk/legacy/aclk_lws_wss_client.h"
"aclk/legacy/aclk_query.c"
"aclk/legacy/aclk_rx_msgs.c"
"aclk/legacy/agent_cloud_link.c"
"aclk/legacy/agent_cloud_link.h"
"aclk/legacy/mqtt.c"
"aclk/legacy/tests/paho-inspection.py"
"backends/backends.c"
"build/m4/ax_gcc_func_attribute.m4"
"build_external/clean-install-arch-debug.Dockerfile"
"build_external/clean-install-arch-extras.Dockerfile"
"build_external/clean-install-arch.Dockerfile"
"build_external/scenarios/aclk-testing/paho-inspection.py"
"collectors/all.h"
"collectors/cgroups.plugin/sys_fs_cgroup.c"
"collectors/charts.d.plugin/charts.d.conf"
"collectors/charts.d.plugin/charts.d.plugin.in"
"collectors/checks.plugin/plugin_checks.c"
"collectors/cups.plugin/cups_plugin.c"
"collectors/diskspace.plugin/plugin_diskspace.c"
"collectors/ebpf.plugin/ebpf.c"
"collectors/ebpf.plugin/ebpf.h"
"collectors/ebpf.plugin/ebpf_apps.c"
"collectors/ebpf.plugin/ebpf_process.c"
"collectors/ebpf.plugin/ebpf_socket.c"
"collectors/freebsd.plugin/freebsd_devstat.c"
"collectors/freebsd.plugin/freebsd_getifaddrs.c"
```

I think we split it like:

- aclk

```cmd
"aclk/legacy/aclk_common.c"
"aclk/legacy/aclk_lws_wss_client.c"
"aclk/legacy/aclk_lws_wss_client.h"
"aclk/legacy/aclk_query.c"
"aclk/legacy/aclk_rx_msgs.c"
"aclk/legacy/agent_cloud_link.c"
"aclk/legacy/agent_cloud_link.h"
"aclk/legacy/mqtt.c"
"aclk/legacy/tests/paho-inspection.py"
```

- backends

```cmd
"backends/backends.c"
```

- build

```cmd
"Dockerfile.test"
"build/m4/ax_gcc_func_attribute.m4"
"build_external/clean-install-arch-debug.Dockerfile"
"build_external/clean-install-arch-extras.Dockerfile"
"build_external/clean-install-arch.Dockerfile"
"build_external/scenarios/aclk-testing/paho-inspection.py"
```

- collectors

```cmd
"collectors/all.h"
"collectors/cgroups.plugin/sys_fs_cgroup.c"
"collectors/charts.d.plugin/charts.d.conf"
"collectors/charts.d.plugin/charts.d.plugin.in"
"collectors/checks.plugin/plugin_checks.c"
"collectors/cups.plugin/cups_plugin.c"
"collectors/diskspace.plugin/plugin_diskspace.c"
"collectors/ebpf.plugin/ebpf.c"
"collectors/ebpf.plugin/ebpf.h"
"collectors/ebpf.plugin/ebpf_apps.c"
"collectors/ebpf.plugin/ebpf_process.c"
"collectors/ebpf.plugin/ebpf_socket.c"
"collectors/freebsd.plugin/freebsd_devstat.c"
"collectors/freebsd.plugin/freebsd_getifaddrs.c"
```

---

@vlvkobal @Ferroin what do you think guys, does it make sense? Would it be better then reviewing this PR?

**Comment by ilyam8** [comment]
Oh, wait, [i forgot](https://github.com/netdata/netdata/pull/10521#issuecomment-813622899) about _30 files per page_ limit. Let me redo the request.

**Comment by ilyam8** [comment]
Another idea 💡 is to move fix spelling in the comments (`# ...`, `// ...`) in a separate PR (easy to review and safe to merge).

**Comment by jsoref** [comment]
I have 122 commits, to filter by comments, I can either flatten or filter I'd have to filter at a commit level (=122 times).

otoh, splitting at the directory/file type level is fairly cheap. So, as long as you're ok w/ directory/file level as a first split, that's definitely my preference.

**Comment by ilyam8** [comment]
Closing in favor of new PRs

**Pull request closed without merging**